documentation/breaking-changes/breaking-changes-tool-help.md

```diff
@@ -0,0 +1,1392 @@
+| 2060 | [Position Changed](#2060---position-changed) |
+| 2070 | [Removed ValueFromPipeline](#2070---removed-valuefrompipeline) |
+| 2080 | [Removed ValueFromPipelineByPropertyName](#2080---removed-valuefrompipelinebypropertyname) |
+| 2090 | [Added ValidateNotNullOrEmpty](#2090---added-validatenotnullorempty) |
+| 2100 | [Removed Parameter From Parameter Set](#2100---removed-parameter-from-parameter-set) |
+| 2110 | [Changed Parameter Element Type](#2110---changed-parameter-element-type) |
+| 2120 | [Changed Parameter Generic Type](#2120---changed-parameter-generic-type) |
+| 2130 | [Changed Parameter Generic Type Argument](#2130---changed-parameter-generic-type-argument) |
+| 2140 | [Different Parameter Generic Type Argument Size](#2140---different-parameter-generic-type-argument-size) |
+| 2150 | [Added ValidateRange](#2150---added-validaterange) |
+| 2160 | [Changed ValidateRange Minimum Value](#2160---changed-validaterange-minimum-value) |
+| 2170 | [Changed ValidateRange Maximum Value](#2170---changed-validaterange-maximum-value) |
+| 3000 | [Changed Property Type](#3000---changed-property-type) |
+| 3010 | [Removed Property](#3010---removed-property) |
+| 3020 | [Changed Element Type](#3020---changed-element-type) |
+| 3030 | [Changed Generic Type](#3030---changed-generic-type) |
+| 3040 | [Changed Generic Type Argument](#3040---changed-generic-type-argument) |
+| 3050 | [Different Generic Type Argument Size](#3050---different-generic-type-argument-size) |
+
+## 1000 - Removed Cmdlet
+
+### Description
+
+_The cmdlet '`<cmdlet>`' has been removed and no alias was found for the original cmdlet name._
+
+When a user can no longer use a cmdlet that was previously available in a module, that is a breaking change. This can occur when the cmdlet is deleted, or the name of the cmdlet was changed with no alias to the original cmdlet name.
+
+### Remediation
+
+_Add the cmdlet '`<cmdlet>`' back to the module, or add an alias to the original cmdlet name._
+
+To add an alias to the original cmdlet name, use the `Alias` attribute for the cmdlet.
+
+```cs
+[Alias("Login-AzureRmAccount")]
+[Cmdlet(VerbsCommon.Add, "AzureRmAccount")]
+public class AddAzureRmAccount : Cmdlet
+{
+    protected override void BeginProcessing()
+    {
+        // cmdlet logic
+    }
+}
+```
+
+## 1010 - Removed Cmdlet Alias
+
+### Description
+
+_The cmdlet '`<cmdlet>`' no longer supports the alias '`<alias>`'._
+
+When a user can no longer use a cmdlet by calling one of its aliases, that is a breaking change. 
+
+### Remediation
+
+_Add the alias '`<alias>`' back to the cmdlet '`<cmdlet>`'._
+
+## 1020 - Changed Output Type
+
+### Description
+
+_The cmdlet '`<cmdlet>`' no longer has output type '`<outputType>`'._
+
+When the output of a cmdlet changes types, that is a breaking change. Existing scripts that assign the output of a cmdlet to a variable and call one of its properties or pass it through to another cmdlet will no longer work.
```

Comment:
I have an issue open to do another refactor to the breaking change tool (slated for the May sprint) that includes this change: https://github.com/Azure/azure-powershell/issues/3507